po/fr.po

```diff
@@ -58,9 +61,9 @@ msgctxt "storage-id-desc"
 msgid "$0 File System"
 msgstr "$0 Système de fichiers"
 
-#: pkg/systemd/overview-cards/usageCard.jsx:84
+#: pkg/systemd/overview-cards/usageCard.jsx:138
 msgid "$0 GiB / $1 GiB"
-msgstr ""
+msgstr "$0 GiB / $1GiB"
```

Comment:
This seems like the issue.

Reply:
Ironically this is fixed in https://github.com/cockpit-project/cockpit/pull/13395 , but we don't want to land that yet. :grin:

Reply:
https://translate.stg.fedoraproject.org/translate/cockpit/master/fr/?q=GiB&checksum=&offset=#history fixed yesterday